tools/pysa_integration_tests/run.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import logging
+import os
+from pathlib import Path
+
+from utils import run_pysa_integration_test, LOG
+
+
+def main() -> None:
+    """
+    Entry point function which checks if full_result.json is there, calls
+    functions from integration_test_utils to run pysa, parse
+    full_result.json, and compare the output.
+    """
+    logging.basicConfig(
+        level=logging.DEBUG,
+        format="%(asctime)s %(levelname)s %(message)s",
+    )
+
+    vulnerable_flask_app_folder = (
+        Path(__file__).resolve().parents[2]
+        / "documentation"
+        / "deliberately_vulnerable_flask_app"
+    )
+
+    full_result_file_path = vulnerable_flask_app_folder / "full_result.json"
+
+    if not os.path.isfile(full_result_file_path):
+        raise FileNotFoundError(
+            f"{full_result_file_path} containing expected issues is not found"
+        )
+
+    LOG.info("Running in `%s`", vulnerable_flask_app_folder)
+    run_pysa_integration_test(
+        vulnerable_flask_app_folder,
```

Comment:
I think the `current_directory` is only used to define in which folder the integration test output is written but not to point pyre to read the `pyre_configuration` file. Does this work if you run `cd tools/pysa_integration_tests && python run.py` ? If it doesn't we can pass `vulnerable_flask_app_folder ` as a passthrough argument with `-l` .

Reply:
You are correct, but with `-l` we require a local configuration, `.pyre_configuration.local`. I will check if there is another work around.

Reply:
Yes we have a workaround with subprocess, if we specify `subprocess.check_output(.., cwd=..)` we can have the desired output, just a bit tests and I shall commit here.